### PR TITLE
Feature/free boxes crop very complex

### PR DESCRIPTION
**Pull request by SebastianInd**
With inflateOccupied(), all infeasible space caused by the size of the robot is set occupied

### CODE REVIEW

File: octomap_world/src/octomap_world.cc [hunk 1]
```diff
@@ -822,6 +822,149 @@ void OctomapWorld::generateMarkerArray(
   }
 }
 
+void OctomapWorld::inflateOccupied() {
```
Comment (helenol) [inline octomap_world/src/octomap_world.cc]: Maybe I'd prefer to have this take in a size to inflate by: inflating stuff by `robot_size_.minCoeff() / 2` isn't very self-evident, and I think it's easier to use if you just give an inflation radius.

Comment (helenol) [reply octomap_world/src/octomap_world.cc]: Hmm although actually I guess you don't inflate it evenly in all directions... But actually consider different sizes in x, y, and z. Ok maybe not.

File: octomap_world/src/octomap_world.cc [hunk 2]
```diff
@@ -822,6 +822,149 @@ void OctomapWorld::generateMarkerArray(
...
+  bbx_center.z() = map_center.z() + map_size.z() / 2 - robot_size_.z() / 4;
+  getKeysBoundingBox(bbx_center, bbx_size, &occupied_keys);
+
+  // Set all infeasible points occupied
+  for (octomap::OcTreeKey key : occupied_keys) {
+    octree_->setNodeValue(octree_->keyToCoord(key), log_odds_value, lazy_eval);
+  }
+
+  // This is necessary since lazy_eval is set to true.
+  octree_->updateInnerOccupancy();
```
Comment (helenol) [inline octomap_world/src/octomap_world.cc]: Maybe stick `if (lazy_eval) {...}` here :) Yes it's a true constant, but if it is ever changes, will be clearer.

File: octomap_world/src/octomap_world.cc (hunk 1 above)
Comment (SebastianInd) [reply octomap_world/src/octomap_world.cc]: Now I take an Eigen::Vector3d as argument to determine the size of inflation ;)

**Comment by SebastianInd** [comment]
Test this please

**Pull request merged**